pkg/query/querier.go

```diff
@@ -1,78 +1,319 @@
 package query
 
 import (
-	"net/http"
-
 	"context"
+	"sync"
+	"unsafe"
+
+	"github.com/go-kit/kit/log"
+	"github.com/prometheus/tsdb"
+	"github.com/prometheus/tsdb/chunks"
+	"github.com/prometheus/tsdb/labels"
 
+	"github.com/improbable-eng/promlts/pkg/store/storepb"
 	"github.com/pkg/errors"
-	"github.com/prometheus/prometheus/pkg/labels"
+	promlabels "github.com/prometheus/prometheus/pkg/labels"
 	"github.com/prometheus/prometheus/promql"
 	"github.com/prometheus/prometheus/storage"
+	"golang.org/x/sync/errgroup"
+	"google.golang.org/grpc"
 )
 
 var _ promql.Queryable = (*Queryable)(nil)
-var _ storage.Querier = (*querier)(nil)
-var _ storage.SeriesSet = (*seriesSet)(nil)
 
 type Queryable struct {
-	client         *http.Client
+	logger         log.Logger
 	storeAddresses []string
 }
 
 // NewQueryable creates implementation of promql.Queryable that uses given HTTP client
```

Comment:
obsolete comment

Reply:
I agree that those comments are kind of tautologies. But golint/govet complains if absent so we went with always adding them in Prometheus upstream.

Would you rather skip those for this project?

Reply:
Well, I am particularly saying here about `that uses given HTTP client` suffix (: Let's just kill this sentence ending and it will be fine.

Reply:
Ah, now I get it :)